Gemfile

```diff
@@ -30,6 +30,10 @@ if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
   gem 'json', '< 2.0.0'
 end
 
+if RUBY_VERSION < '2.0.0' && !!(RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/)
+  gem 'ffi', '< 1.9.15' # allow ffi to be installed on older rubies on windows
+end
```

Comment:
Its not but its a requirement of getting the build to pass. It was done here first as a test and has since been merged upstream; but I didn't think it was worth rebasing just to remove it from the github UI.